.vib/ruby/goss/ruby.yaml

```
@@ -0,0 +1,16 @@
+command:
+  check-installed-gems:
+    exec: gem list
```

Ensure you are only listing **local** installed gems
```suggestion
    exec: gem list --local
```